### PR TITLE
Populate Repometrics.json with Cookiecutter

### PR DESCRIPTION
**Pull request by RicardoZamora01**
## Problem

Currently, we want to add metrics to the repo reports that include where it is hosted, project fisma level, project type, etc. We will be using this to create tags in the reports and help create visualizations.

## Solution

Created a cookicutter template that asks multiple choice and free response questions, the repometrics.json will be generated inside the project that is created by cookiecutter and moved to parent directory.

### CODE REVIEW

**Comment by RicardoZamora01** [comment]
PR for #97

File: tier2/repometrics/hooks/post_gen_project.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import shutil
+import time 
+
+# Get working directory and repometrics file
+current_dir = os.getcwd()
+repometrics_file = "repometrics.json"
+source_file = os.path.join(current_dir, repometrics_file)
+
+# Get parent directory, one level up from the current directory
+parent_dir = os.path.abspath(os.path.join(current_dir, '..'))
```
Comment (decause-gov) [inline tier2/repometrics/hooks/post_gen_project.py:11]: I'm not an expert, but I believe this is a reasonable method, and doesn't open us up to path-traversal attacks, but maybe @IsaacMilarky can take a closer look.

**Review by decause-gov — COMMENTED** [review]
Other than my question about path-traversal attacks, things look pretty decent to me.

Have you tested running this on the commandline yet? 
Once we like it for Tier2, we should consider pushing this to Tier3 and Tier4 as well.

I'll leave the final reviews to @natalialuzuriaga and @IsaacMilarky

File: tier2/{{cookiecutter.project_slug}}/repometrics/hooks/post_gen_project.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+# Remove directory once repometrics file was successfully moved
+os.chdir("..")
+dir_name = "{{cookiecutter.project_type}}"
+
+try:
+    # Check if the directory exists
+    if os.path.exists(dir_name):
+        # Attempt to remove the directory
+        shutil.rmtree(dir_name)
```
Comment (RicardoZamora01) [inline tier2/{{cookiecutter.project_slug}}/repometrics/hooks/post_gen_project.py:35]: After reviewing my own code, I realized that I deleted the command itself at some point 😅

**Comment by IsaacMilarky** [comment]
This branch seems to be kind of messed up at the moment. I would suggest either cherry picking your changes or trying to redo the commits entirely as the easiest way of not having to deal with git nonsense. Force pushing usually isn't a good idea unless there isn't a better way of doing things. 

Also, you should be able to do draft PRs if you want feedback for changes that aren't quite ready to merge.

**Comment by RicardoZamora01** [comment]
closed due to misalignments in dev and main

**Pull request closed without merging**